.github/workflows/unitTests.yaml

```diff
@@ -64,8 +64,6 @@ jobs:
           install
 
     - name: Unit Tests
-      id: unitTest1
-      continue-on-error: true
```

Comment:
What about other places where you have `continue-on-error: true`?

Reply:
Those should be OK - this was accidentally left over from when I was retrying unit tests on failure.

Reply:
ah ok